tests/test_schemaview.py

```diff
@@ -0,0 +1,16 @@
+"""Test for sorting MappingSetDataFrame columns."""
+import unittest
+
+from sssom.constants import SSSOMSchemaView
+
+
+class TestSort(unittest.TestCase):
+    """A test case for sorting msdf columns."""
+
+    def setUp(self) -> None:
+        """Test up the test cases with the third basic example."""
+        self.sv = SSSOMSchemaView()
+
+    def test_slots(self) -> None:
+        """Test slots."""
+        self.assertEqual(len(self.sv.mapping_slots), 39)
```

Comment:
probably better test `self.assertTrue(len(self.sv.mapping_slots)>=39)` because changes to the metadata model will break this test every time?

Reply:
Good call, changed!